tests/contrib/uwsgi/test_uwsgi.py

```diff
@@ -0,0 +1,35 @@
+import os
+import subprocess
+import shlex
+
+current_dir = os.path.dirname(__file__)
+
+def test_uwsgi_threads_enabled():
+    command = "uwsgi --http :9090 --wsgi-file {} --enable-threads --log-master".format(
+        os.path.join(current_dir, "uwsgi_threads.py")
+    )
+    out = subprocess.check_output(shlex.split(command))
```

Comment:
might be worth factoring this out, but not a big deal.

Reply:
> This overall looks like a good approach to me. I don't see any integration in our code flow, I imagine the idea is for us to pick this up and integrate it somewhere in the tracer startup code?

Right, I've tested this logic placing it just after this line in order to have it run once the first trace is sent and the necessary thread is created for the tracer to run:
https://github.com/DataDog/dd-trace-py/blob/master/ddtrace/_worker.py#L52
If we decided to stop the tracer, I imagine we could do so within the function by adding the _workers.py `stop()` function after logging the error.

@brettlangdon Mentioned potentially using it at some point in the tracer import stages so we decided to make it a utility function to make it more flexible.  There also may need to be other uWSGI config checks added in the future arrangement should help group them logically.

Reply:
> might be worth factoring this out, but not a big deal.

Made some adjustments to factor out some variables and a function for checking output.